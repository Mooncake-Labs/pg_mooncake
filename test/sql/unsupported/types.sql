CREATE TABLE t (a jsonb) USING columnstore;

CREATE TYPE point AS (x int, y int);
CREATE TABLE t (a point) USING columnstore;

CREATE TABLE mooncake_numeric_tbl (val numeric(40, 5)) USING columnstore;
CREATE TABLE mooncake_numeric_tbl (val numeric) USING columnstore;
