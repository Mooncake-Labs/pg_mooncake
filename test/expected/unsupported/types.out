CREATE TABLE t (a jsonb) USING columnstore;
ERROR:  column "a" has unsupported type
CREATE TYPE point AS (x int, y int);
CREATE TABLE t (a point) USING columnstore;
ERROR:  column "a" has unsupported type
CREATE TABLE mooncake_numeric_tbl (val numeric(40, 5)) USING columnstore;
ERROR:  Unsupported type when creating column: type precision 40 with scale 5 is not supported by duckdb, which only allows maximum precision 38
CREATE TABLE mooncake_numeric_tbl (val numeric) USING columnstore;
ERROR:  Unsupported type when creating column: type precision 65535 with scale -5 is not supported by duckdb, which only allows maximum precision 38
